compiler-core/src/language_server/tests.rs

```diff
@@ -559,6 +559,7 @@ impl<'a> TestProject<'a> {
     pub fn positioned_with_io(
         &self,
         position: Position,
+        new_src: Option<&str>,
```

Comment:
Remove this please. The other tests should not change, we don't want to have a parameter that every test needs to provide but only this one uses.